index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script src="dist/checker.js"></script>
+</head>
+<body>
+    Some content that should not appear on unsupported browsers
```

Comment:
can you please remove this content since it's displayed, it's a bit misleading
![image](https://user-images.githubusercontent.com/468620/112011774-e14ffb00-8b28-11eb-8616-c769d5d5bbf1.png)

Reply:
Checked with IE10 useragent on Mac OS Chrome. New version works fine to me

Please recheck it in your environment.

Can be checked by using this [link](https://htmlpreview.github.io/?https://raw.githubusercontent.com/oat-sa/browser-support-checker/feature/RO-233/browser-support-checker/index.html)